bndtools.builder/src/org/bndtools/builder/BuilderPlugin.java

```diff
@@ -1,12 +1,24 @@
 package org.bndtools.builder;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map.Entry;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.ui.IDecoratorManager;
+import org.eclipse.ui.plugin.AbstractUIPlugin;
 import org.osgi.framework.BundleContext;
 
-public class BuilderPlugin extends org.eclipse.core.runtime.Plugin {
+public class BuilderPlugin extends AbstractUIPlugin {
```

Comment:
Without using AbstractUIPlugin, line 51 fails to compile:

https://github.com/wodencafe/bndtools/blob/fc629d8dd67a7665dc213db694629aa5e3b31807/bndtools.builder/src/org/bndtools/builder/BuilderPlugin.java#L47-L56